src/importer.cpp

```diff
@@ -621,6 +621,7 @@ bool Importer::resolveImports(ModelPtr &model, const std::string &basePath)
     bool status = true;
     History history;
 
+    pFunc()->removeAllIssues();
```

Comment:
`resolveImport` documentation should be updated to note this behaviour

Reply:
0d138f4